src/harness/reference_models/pre_iap/pre_iap.py

```diff
@@ -0,0 +1,55 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+  This is the Pre-IAP reference model which implements calling of 4 sub reference
+  modules in to ensure the checks to be done before sending to the IAP calculation.
+==================================================================================
+"""
+
+from reference_models.fss_purge import fss_purge
+from reference_models.inter_sas_duplicate_grant import inter_sas_duplicate_grant
+
+
+def pre_iap(protected_entities, uut_fad_object, test_harness_fad_object, headroom):
```

Comment:
`preIapFilteringReferenceModel` or `performPreIapFiltering`.

Reply:
Changed   to performPreIapFiltering